src/helpers.js

```diff
@@ -42,29 +42,28 @@ const getCourseSectionFromFeatureUrl = courseFeature => {
   return urlParts[urlParts.length - 1].split("#")[0]
 }
 
-const getCourseCollectionText = courseCollection => {
+const getCourseCollectionObject = courseCollection => {
   const feature = courseCollection["ocw_feature"]
   const subfeature = courseCollection["ocw_subfeature"]
-  const specialty = courseCollection["ocw_specialty"]
-  let collection = feature
+  const speciality = courseCollection["ocw_speciality"]
+  const collection = {}
+  if (feature) {
+    collection["topic"] = feature
+  }
   if (subfeature) {
-    collection = `${collection} > ${subfeature}`
+    collection["subtopic"] = subfeature
   }
-  if (specialty) {
-    collection = `${collection} > ${specialty}`
+  if (speciality) {
+    collection["speciality"] = speciality
```

Comment:
are we thinking we want to stick with the `speciality` spelling going forward? I guess we probably don't want to mix them...

Reply:
question for @pdpinch too I think

Reply:
I don't have any strong opinions about the spelling itself.  As you said: tomato, tomahto.  However, I vote that we should keep it `speciality` to be consistent with the way it is in master.json so that if someone in the future is trying to look back at the source data they will be less confused maybe.

Reply:
sounds good to me